configs/retinaface/retinaface_r152_fpn_ssh_widerface.py

```diff
@@ -0,0 +1,165 @@
+_base_ = [
+    '../_base_/default_runtime.py',
+]
+
+dataset_type = 'WIDERFaceDataset'
+data_root = 'data/WIDERFace/'
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[1, 1, 1], to_rgb=True)
+data = dict(
+    samples_per_gpu=1,
+    workers_per_gpu=1,
+    train=dict(
+        type=dataset_type,
+        ann_file=data_root + 'WIDER_train/train.txt',
+        img_prefix=data_root + 'WIDER_train/',
+        min_size=1,
+        offset=0,
+        pipeline=[
+            dict(type='LoadImageFromFile', to_float32=True),
+            dict(type='LoadAnnotations', with_bbox=True),
+            dict(type='RandomSquareCrop', crop_ratio_range=[0.3, 1.0]),
+            dict(
+                type='PhotoMetricDistortion',
+                brightness_delta=32,
+                contrast_range=(0.5, 1.5),
+                saturation_range=(0.5, 1.5),
+                hue_delta=18,
+                saturate=True),
+            dict(type='RandomFlip', flip_ratio=0.5),
+            dict(type='Resize', img_scale=(640, 640), keep_ratio=False),
+            dict(type='Normalize', **img_norm_cfg),
+            dict(type='DefaultFormatBundle'),
+            dict(
+                type='Collect',
+                keys=['img', 'gt_bboxes', 'gt_labels', 'gt_bboxes_ignore']),
+        ]),
+    val=dict(
+        type=dataset_type,
+        ann_file=data_root + 'WIDER_val/val.txt',
+        img_prefix=data_root + 'WIDER_val/',
+        min_size=1,
+        offset=0,
+        pipeline=[
+            dict(type='LoadImageFromFile'),
+            dict(
+                type='MultiScaleFlipAug',
+                img_scale=(640, 640),
+                flip=False,
+                transforms=[
+                    dict(type='Resize', keep_ratio=True),
+                    dict(type='RandomFlip', flip_ratio=0.0),
+                    dict(type='Normalize', **img_norm_cfg),
+                    dict(type='Pad', size_divisor=32, pad_val=0),
+                    dict(type='ImageToTensor', keys=['img']),
+                    dict(type='Collect', keys=['img'])
+                ])
+        ]),
+    test=dict(
+        type=dataset_type,
+        ann_file=data_root + 'WIDER_val/val.txt',
+        img_prefix=data_root + 'WIDER_val/',
+        # min_size=1,
+        offset=0,
+        pipeline=[
+            dict(type='LoadImageFromFile'),
+            dict(
+                type='MultiScaleFlipAug',
+                img_scale=[(500, 750), (800, 1200), (1100, 1650), (1400, 2100),
```

Comment:
It's better to provide single scale testing config for easy use. And the img_scale in val and test dataset are not the same.